### PR TITLE
quickstart enhancement

### LINKED ISSUE #595: Fun in 5 minutes tutorial?
@technosophos asked me to file this.

Can I get a tutorial that tells one how to:
1. Install brigade
2. Install Kashti
3. Do something with brigade and see the result

All in 5 minutes or less on average.

### PR DESCRIPTION
**Pull request by dgkanatsios**
**What this PR does / why we need it**:

'Modernized' Brigade quickstart. Closes #595, partially #748  

For easy viewing, check here: https://github.com/dgkanatsios/brigade/blob/quickstart/README.md#quickstart

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,77 +32,164 @@ architecture.
...
+[brigade:k8s] Pod not yet scheduled
+[brigade:k8s] default/do-nothing-01d0y7bcxs6ke0yayrx6nbvm39 phase Pending
+[brigade:k8s] default/do-nothing-01d0y7bcxs6ke0yayrx6nbvm39 phase Succeeded
+done
+```
+
+As you can see, Brigade created a new pod for this Build (called `do-nothing-01d0y7bcxs6ke0yayrx6nbvm39`) that executed our job. Let's get its logs.
+
+```bash
+kubectl logs do-nothing-01d0y7bcxs6ke0yayrx6nbvm39
```
Comment (radu-matei) [inline README.md]: Was there a PR at some point that follows the logs directly?
Can we use that?

Comment (dgkanatsios) [reply README.md]: yes, but it's not still in the released version of `brig`. Taking it as a task to update it when we do that.

File: README.md [hunk 2]
```diff
@@ -32,77 +32,164 @@ architecture.
...
+...other cluster Secrets...
 ```
 
-The Helm chart will be removed before Brigade 1.0.0, so we recommend using
-`brig project create` instead. Note that the projects created with the Helm chart
-remain compatible with Brigade.
+### Creating Your First Brigade Build
 
-### Creating Your First `brigade.js`
+Conceptually, a Brigade Build is a set of instructions (written in JavaScript, in a file called `brigade.js`) that will run upon a triggering of an event. Event can be something like a git push, a Docker push or just a webhook. In this way, you can think of Builds like job(s)/task(s)/pipeline(s).
```
Comment (vdice) [inline README.md]: Pluralization needed in `Events can be something like....`

Comment (vdice) [reply README.md]: Perhaps `... you can think of Builds as jobs/tasks/pipelines`

File: README.md [hunk 3]
```diff
@@ -32,77 +32,164 @@ architecture.
...
 
-Creating your first `brigade.js` is as easy as this:
+Let's now create a new file called `brigade.js` with the following content:
 
 ```javascript
-const { events } = require("brigadier");
+const { events, Job } = require("brigadier")
```
Comment (vdice) [inline README.md]: To follow js conventions, perhaps we add `;` to all statements.

File: README.md [hunk 4]
```diff
@@ -32,77 +32,164 @@ architecture.
...
+You will now have Brigade installed. [Kashti](https://github.com/Azure/kashti), the dashboard for your Brigade pipelines, is also installed in the cluster.
 
-In your local environment, install `brig`, the Brigade commandline client. To get
-the latest version, go to the [releases page](https://github.com/Azure/brigade/releases/)
-and download the binary for your platform. Alternatively, you can use
-[asdf-brig](https://github.com/Ibotta/asdf-brig) to install & manage multiple
-versions of `brig`.
+### Install brig
+
+Brig is the Brigade command line client. You can use `brig` to create/update/delete new brigade Projects, run Builds etc. To get `brig`, navigate to the [Releases page](https://github.com/Azure/brigade/releases/) and then download the appropriate client for your platform. For example, if you're using Linux or WSL, you can get the 0.19.0 version in this way:
```
Comment (vdice) [inline README.md]: nit: missing comma `.. run Builds, etc.`

**Comment by dgkanatsios** [comment]
thanks both, pushed a new commit with corrections suggested by @vdice

**Pull request merged**